docs/content/docs/stimulus-chartjs.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Chartjs
 description: A Stimulus controller to deal with chart.js.
-package: chart
+package: chartjs
```

Comment:
Nice catch!

But it's incomplete, this value is used in the installation section and will change 

```diff
- import Chart from '@stimulus-components/chartjs'
+ import Chartjs from '@stimulus-components/chartjs'

- application.register('chart', Chart)
+ application.register('chartjs', Chartjs)
```

Which makes the others sections obsolete, can you update them as well?

Thanks! 🙏

Reply:
Done!